lib/logging.js

```diff
@@ -159,6 +160,33 @@
             }
         };
     }
+    
+    function makeTimerStartMethod() {
+        return function (timerName) {
+            if (!timerName) {
+                return;
+            }
+            if (!this._timerData) {
+                this._timerData = {};
+            }
+            var startDate = new Date();
+            this._timerData[timerName] = startDate;
```

Comment:
It seems like this might be read better if `_timerData` were defined more clearly on the Logger object.  In any case, I think you need to clean up this element in `End` function after it is retrieved.